packages/admin-app/src/components/GuardianTable/GuardianTable.tsx

```diff
@@ -37,15 +44,17 @@ const columns: GridColDef[] = [
 export const GuardianTable: React.FC<GuardianTableProps> = ({ data }) => {
     const classes = useStyles();
     return (
-        <Box display="flex" minHeight="500px" height="100%" width="100%">
-            <DataGrid
+        <Box display="flex" height="100%" width="100%">
```

Comment:
Hmm. This def seems like there should be a general grid based on the `StyledDataGrid`. Maybe some kind of wrapper.